src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+

```

Comment:
same

Reply:
u should be kidding?
im, like all time, used command -mvn clean package and anything correct, but if I guided of your recommendation its incorrect ;//